test/quantization/test_quantize_script.py

```diff
@@ -1100,7 +1101,10 @@ class TestQuantizeScriptPTSQOps(JitTestCase):
     for individual ops end to end.
     """
     def _test_op_impl(self, module, data, quantized_op):
-        qconfig_dict = {'': get_default_qconfig('fbgemm')}
+        qengine = torch.backends.quantized.engine
+        if qengine == 'none':
+            qengine = 'fbgemm'
```

Comment:
why not pass in qengine here?

Reply:
We would still have to check for `none` case for OS like macos where neither is enabled to get the correct qconfig.

Reply:
can we just add this to get_default_qconfig?

Reply:
yeah I was planning to do it in a follow up PR

Reply:
See #38407